app/src/main/res/layout/activity_home.xml

```diff
@@ -9,16 +9,12 @@
     android:orientation="vertical"
     tools:context=".HomeActivity">
 
-    <androidx.appcompat.widget.Toolbar
-        android:id="@+id/navigationToolbar"
+    <ViewStub
+        android:id="@+id/navigationToolbar_stub"
+        android:inflatedId="@id/navigationToolbar_stub"
```

Comment:
nit: -> navigationToolbarStub

Side note: I don't love that the inflated ID is still called "stub" but I think it prevents people from accidentally using it, thinking it's the full toolbar.